test/units/testsuite-50.sh

```diff
@@ -41,6 +41,14 @@ systemd-dissect --json=short "${image}.raw" | grep -q -F '{"rw":"ro","designator
 systemd-dissect "${image}.raw" | grep -q -F "MARKER=1"
 systemd-dissect "${image}.raw" | grep -q -F -f <(sed 's/"//g' "$os_release")
 
+systemd-dissect --list "${image}.raw" | grep -q '^etc/os-release$'
+
+read SHA256SUM1 GARBAGE < <(systemd-dissect --copy-from "${image}.raw" etc/os-release | sha256sum)
+test "$SHA256SUM1" != ""
+read SHA256SUM2 GARBAGE < <(systemd-dissect --read-only --with "${image}.raw" sha256sum etc/os-release)
```

Comment:
```suggestion
read -r SHA256SUM2 _ < <(systemd-dissect --read-only --with "${image}.raw" sha256sum etc/os-release)
```

ditto

Reply:
Not sure how I managed to post this one outside of the shellcheck warning comment, oh well.